main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"final_project/auth"
+	db "final_project/database"
+	"final_project/handlers"
+	"final_project/tests"
+
+	"github.com/joho/godotenv"
+)
+
+// startServer() обеспечивает начало работы сервера, создание БД и настройку API
+func startServer() {
+	ports := ":" + strconv.Itoa(tests.Port)
+	webDir := "./web"
+
+	if p := os.Getenv("PORT"); p != "" {
+		ports = ":" + p
+	}
+
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("ошибка загрузки .env файла")
```

Comment:
Хорошо бы выводить ошибку, а то так и не узнаем в чем проблема
```suggestion
		log.Fatal("ошибка загрузки .env файла", err)
```